lib/LaTeXML/Engine/LaTeX.pool.ltxml

```diff
@@ -4889,6 +4889,7 @@ DefConstructor('\@framebox[Dimension][]{}',
           $document->setAttribute($c[0], $k => $v); } } } }
 );
 
+AssignValue(allocated_boxes => 0);
```

Comment:
Maybe the AssignValue should be `'global'` scoped? Likely identical, until it isn't :>

Reply:
will do.